install_config/aggregate_logging.adoc

```diff
@@ -126,8 +130,9 @@ removes all logging persistent data including PVC, set
 *eventrouter* will be installed. When both are `false`, *eventrouter* will be
 uninstalled.
 
-|`openshift_logging_eventrouter_image_prefix`
-|The prefix for the *eventrouter* logging image.
+|`openshift_logging_eventrouter_image`
+|The image version for Eventrouter.
+For example: `openshift_logging_eventrouter_image=<registry_url>:5000/openshift3/logging-eventrouter:v3.X.X`
```

Comment:
Shall we use the offical image?   registry.redhat.io/openshift3/ose-logging-eventrouter:v3.11